codeworld-requirements/src/RequirementsChecker.hs

```diff
@@ -0,0 +1,37 @@
+module RequirementsChecker (plugin) where
```

Comment:
Google's lawyers require that new source code has an appropriate copyright header, which you can copy from any other file in the project.

Reply:
I've now fixed these legal/copyright issues.